SR2025/2024-09-02-competition-announcement.md

```diff
@@ -0,0 +1,35 @@
+---
+to: Potential team supervisors
+subject: Announcing Student Robotics 2025!
+---
+
+Hi,
+
+We're pleased to [announce][announcement] that registration for Student Robotics
+2025 is now open. If you would like to compete, please complete the
+[signup form][signup-form] with the details of your team. All future emails
+about SR2025 will be sent to the details you provide there.
+
+This year we're hosting [Kickstart][kickstart] at the University of Southampton,
+on Saturday 12th October. We will also be live-streaming the event for those
+unable to attend in person. We expect to confirm places towards the latter half
+of September. Places are limited, so sign up soon to avoid disappointment.
+
+As usual we will loan your team a [kit][kit] of parts to help build your robot.
+This year we hope to send you your kit before Kickstart. We will also be sharing
+some [pre-Kickstart activities][pre-kickstart-activities] to enable your team to
+start learning about the kit.
```

Comment:
TODO:
- the activities are about robotics and the simulator, not the kit
- we're publishing them at the same time as this email goes out, so the tenses here should be updated

Reply:
e443d9f